xml/obs_build_process.xml

```diff
@@ -53,33 +54,34 @@
        KVM
       </para>
      </listitem>
+     <listitem>
+      <para>
+       Qemu
+      </para>
+     </listitem>
     </itemizedlist> 
     <para>
     In the preinstall phase, the OBS Worker creates a small
-    base system (chroot or VM Image) with manually extracted packages (file
-    system, coreutils, binutils, rpm/debutils, etc.), copies all necessary
-    build requirements into the base system. </para>
+    base system out of the packages declared to be preinstalled (file
+    system, coreutils, binutils, rpm/debutils, etc.). These must be just
+    enough to install further packages. In addition it copies all necessary
```

Comment:
"These must be just enough to install further packages." is a bit unclear -> suggestion: "The tools installed in this phase must only provide the minimum functionality necessary to allow installing further packages."